packages/mui-joy/src/styles/sxConfig.ts

```diff
@@ -0,0 +1,35 @@
+import { unstable_defaultSxConfig } from '@mui/system';
+
+const sxConfig = {
```

Comment:
For comparison, this is how it looked previously: https://github.com/mui/material-ui/blob/1c897f813aea4f3293270071aa3851cc214e0da9/packages/mui-joy/src/styles/styleFunctionSx.ts